docs/architecture/third-party/drupal/content-architecture.md

```diff
@@ -1 +1,14 @@
--- TO BE DEFINED --
\ No newline at end of file
+# Content architecture
+
+The OpenEuropa project does not enforce a strict content architecture but provides corporate solutions used by the European Commission.
+
+## Content types
+
+The [OpenEuropa Content](https://github.com/openeuropa/oe_content) component provides a growing number of corporate content types intended to harmonize the content architecture across the sites of the European Commission.
+
+Moreover, it provides RDF-based support for the corporate vocabularies published by the European Publication Office (OP).
```

Comment:
Can we add the link of the European Publication Office?
e.g. https://publications.europa.eu/en/home